libs/examples/src/filter-field/filter-field-async-example/filter-field-async-example.ts

```diff
@@ -55,7 +55,12 @@ export class DtExampleFilterFieldAsync {
   currentFilterChanged(
     event: DtFilterFieldCurrentFilterChangeEvent<any>,
   ): void {
-    if (event.added[0] === this.DATA.autocomplete[0]) {
+    if (
+      event.added[0] === this.DATA.autocomplete[0] ||
+      this.ASYNC_DATA.autocomplete.find((name) => {
+        return name === event.removed[0];
```

Comment:
Look good to me. I think you can even simplify it to one line with `find(name => name === event.removed[0])`  if you want.

Reply:
It threw an error originally but that was due to me adding the curly brackets by mistake. Thank you!

Reply:
Gladly :)